DOCUMENTATION.md

```diff
@@ -30,7 +45,7 @@ MQTTClient(app_id, app_access_key, [mqtt_address], [discovery_address])
 - `app_access_key`: **string**  this can be found at the bottom of the application page under **ACCESS KEYS**.
 ![Screenshot of the console with accesskey section](./images/accesskey-console.png?raw=true)
 - `mqtt_address`: **string**  this  is the address of the handler to which the application was registered. It needs to be provided as an `mqtt_address=value` argument when calling the constructor.
-- `discovery_address`: **string** this is the address of the discovery server to use in order to find back the address of the MQTT handler. It needs to be provided as an `discovery_address=value` argument when calling the constructor.
+- `discovery_address`: **string**  this is the address of the discovery server to use in order to find back the address of the MQTT handler. It needs to be provided as an `discovery_address=value` argument when calling the constructor.
```

Comment:
Same, that it's the name argument `discovery_address` can be guessed, no need for the second sentence